op-chain-ops/upgrades/l1.go

```diff
@@ -305,30 +305,90 @@ func L2OutputOracle(batch *safe.Batch, implementations superchain.Implementation
 		return err
 	}
 
-	var l2OutputOracleStartingBlockNumber, l2OutputOracleStartingTimestamp *big.Int
+	l2OutputOracle, err := bindings.NewL2OutputOracleCaller(common.HexToAddress(list.L2OutputOracleProxy.String()), backend)
+	if err != nil {
+		return err
+	}
+
+	l2OutputOracleSubmissionInterval, err := l2OutputOracle.SubmissionInterval(&bind.CallOpts{})
+	if err != nil {
+		return err
+	}
+
+	l2BlockTime, err := l2OutputOracle.L2BlockTime(&bind.CallOpts{})
+	if err != nil {
+		return err
+	}
+
+	l2OutputOracleStartingBlockNumber, err := l2OutputOracle.StartingBlockNumber(&bind.CallOpts{})
+	if err != nil {
+		return err
+	}
+
+	l2OutputOracleStartingTimestamp, err := l2OutputOracle.StartingTimestamp(&bind.CallOpts{})
+	if err != nil {
+		return err
+	}
+
+	l2OutputOracleProposer, err := l2OutputOracle.Proposer(&bind.CallOpts{})
+	if err != nil {
+		return err
+	}
+
+	l2OutputOracleChallenger, err := l2OutputOracle.Challenger(&bind.CallOpts{})
+	if err != nil {
+		return err
+	}
+
+	finalizationPeriodSeconds, err := l2OutputOracle.FinalizationPeriodSeconds(&bind.CallOpts{})
+	if err != nil {
+		return err
+	}
+
 	if config != nil {
-		l2OutputOracleStartingBlockNumber = new(big.Int).SetUint64(config.L2OutputOracleStartingBlockNumber)
+		if l2OutputOracleSubmissionInterval != new(big.Int).SetUint64(config.L2OutputOracleSubmissionInterval) {
```

Comment:
I recommend using u64s for comparison over big.Ints for simplicity

`l2OutputOracleSubmissionInterval.Uint64()`